nntrainer/meson.build

```diff
@@ -56,14 +56,12 @@ foreach elem : nntrainer_elements
   nntrainer_inc_abs += meson.current_source_dir() / elem
 endforeach
 
-# add third party to include path
-fs = import('fs')
-if fs.is_dir('../third_party/ruy') 
-  nntrainer_inc += include_directories('../third_party/ruy')
-  nntrainer_inc_abs += meson.source_root() / 'third_party' / 'ruy'
-else
-  error('The git submodules of the nntrainer repository have not been checked out.\n Try: git submodule update --init')
-endif
+# Configure the Ruy project (CMake)
+cmake = import('cmake')
+ruy_options = cmake.subproject_options()
+ruy_options.add_cmake_defines({'RUY_MINIMAL_BUILD': true})
+ruy_proj = cmake.subproject('ruy', options: ruy_options, required: true)
+ruy_dep = ruy_proj.dependency('ruy')
```

Comment:
If you would like to keep it as includes only only dependency, it'd be:
```meson
ruy_dep = ruy_proj.dependency('ruy').partial_dependency(includes: true)
```
That would keep it as it were.